classes/class-wc-connect-nux.php

```diff
@@ -452,50 +450,26 @@ public function show_banner_after_connection() {
 
 			$this->tracks->opted_in( 'connection_banner' );
 
-			$this->show_nux_banner( array(
-				'title'          => __( 'Setup complete! You can now enjoy discounted shipping rates and print labels directly from your dashboard.', 'woocommerce-services' ),
-				'description'    => esc_html( __( 'When you’re ready, you can purchase discounted labels from USPS, and print USPS labels at home.', 'woocommerce-services' ) ),
-				'button_text'    => __( 'Got it, thanks!', 'woocommerce-services' ),
-				'button_link'    => add_query_arg( array(
-					'wcs-nux-notice' => 'dismiss',
-				) ),
-				'image_url'      => plugins_url(
-					'images/nux-printer-laptop-illustration.png', dirname( __FILE__ )
-				),
-				'should_show_jp' => false,
-				'should_show_terms' => false,
-			) );
-		}
-
-		public function show_tos_banner() {
-			if ( isset( $_GET['wcs-nux-tos'] ) && 'accept' === $_GET['wcs-nux-tos'] ) {
-				WC_Connect_Options::update_option( 'tos_accepted', true );
+			$country = WC()->countries->get_base_country();
 
-				$this->tracks->opted_in( 'tos_banner' );
-
-				wp_safe_redirect( remove_query_arg( 'wcs-nux-tos' ) );
-				exit;
+			if ( 'US' === $country ) {
+				$description = __( 'You can now enjoy automated tax calculation, live shipping rates, shipping label printing, and smoother payment setup.', 'woocommerce-services' );
+			} else {
+				$description = __( 'You can now enjoy automated tax calculation, live shipping rates, and smoother payment setup.', 'woocommerce-services' );
 			}
 
 			$this->show_nux_banner( array(
-				'title'          => __( 'Almost ready to enjoy discounted shipping rates', 'woocommerce-services' ),
-				'description'    => sprintf( wp_kses( __( 'Everything is ready to roll, we just need you to agree to our <a href="%1$s">Terms of Service</a>.', 'woocommerce-services' ),
-					array(
-						'a' => array(
-							'href' => array(),
-						),
-					) ),
-					'https://woocommerce.com/terms-conditions/'
-				),
-				'button_text'    => __( 'I accept', 'woocommerce-services' ),
+				'title'          => __( 'Setup complete.', 'woocommerce-services' ),
```

Comment:
This title has no punctuation in other past or current cases — may want to omit?

Reply:
Ah, as the cleaned up diff makes clear, I was incorrect about punctuation not being used at the end of titles.